website/docs/d/application.html.markdown

```diff
@@ -7,7 +7,7 @@ description: |-
 ---
 
 # Data Source: newrelic\_application
-~> **DEPRECATED** Use at your own risk. Use the [`newrelic_entity`](/docs/providers/newrelic/d/entity.html) data source instead. This feature may be removed in the next major release.
+~> **DEPRECATED** Use at your own risk. Use the [`newrelic_entity`](../../docs/data-sources/entity) data source instead. This feature may be removed in the next major release.
```

Comment:
```suggestion
~> **DEPRECATED** Use at your own risk. Use the [`newrelic_entity`](/providers/newrelic/newrelic/latest/docs/data-sources/entity) data source instead. This feature may be removed in the next major release.
```

I'd use absolute url (`https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/entity`), but in sake of consistency (e.g. #1659) you could omit `https://registry.terraform.io` part.